chart/kubeapps/templates/custom-component.pvc.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
```

Comment:
Shouldn't this PVC be conditional on the user setting some custom component in the chart?

Reply:
In fact, I would use a configmap instead. At the moment it seems that we are just defining a single file `custom_components.js` so I would follow the same approach than `custom_style.css` which is adding the code in the `dashboard-config.yaml` file (and mount it from the config file):

```
  custom_components.js: |-
{{- .Values.dashboard.customComponents | nindent 4 }}
```

Reply:
@andresmgot are you sure placing a js bundle into a configmap is the route that you want to go? a bundle can get rather large

Reply:
> In fact, I would use a configmap instead. 

Yep, I'd suggested the same in my [main comment](https://github.com/kubeapps/kubeapps/pull/2483#pullrequestreview-601342624). 

> @andresmgot are you sure placing a js bundle into a configmap is the route that you want to go? a bundle can get rather large

Do you mean that a bundle of custom components for the Kubeapps might be > 1MB ? I'd be pretty horrified if that's the case :) Or do you mean that you think there would be an issue with a 10k bundle in a configmap?

Reply:
I've just never used a configMap like this before but since this is the established pattern I should follow it 👍